_episodes/04-building-containers.md

```diff
@@ -27,6 +27,48 @@ and then deploy the container into a production environment for execution (as yo
   <figcaption>'Singularity usage workflow' via <i>Kurtzer GM, Sochat V, Bauer MW (2017) Singularity: Scientific containers for mobility of compute. PLoS ONE 12(5): e0177459. <a href="https://doi.org/10.1371/journal.pone.0177459">https://doi.org/10.1371/journal.pone.0177459</a></i></figcaption>
 </figure>
 
+# Hello World Singularity
+
+The following recipe shows how to build a hello-world container, and run the container on your local computer.
+
+- Step 1: Open a text editor
+
+  ~~~
+  gedit hello-world.def
```

Comment:
Should we use `nano` here? In case people run this on a server?

Maybe also add "a text editor (e.g., `nano`, `vim`, or `gedit` in a graphical environment)"

Reply:
done